examples/bri-2/README.md

```diff
@@ -54,3 +28,26 @@ npm run dev
 ```
 
 Navigate to `http://localhost:3000` on your web browser to view the `dashboard`.
+
+# Troubleshooting
+
+- If you have an existing bri-2 build, run the following sequence to remove old build artifacts:
+```
+make clean
+make build
+make start
+```
+- You may need to run `make build` twice in order to properly compile smart contracts
+- Note: Environment variables default to use Ganache as the Ethereum network
+
+# Architecture
+
+Here is a comparison of the reference implementations:
+
+| Service Type | bri-1 | bri-2 |
+| -------- | ----- | ----------- |
+| Eth. client | `Nethermind` | `commit-mgr` + `mongo` + `ITX` |
```

Comment:
BRI-1 should say NChain instead of Nethermind here. Nethermind uses NChain also on the way "out" of the stack to layer 1.